requirements.txt

```diff
@@ -1 +1 @@
-importlib_metadata>=1.7.0;python_version<'3.8'
+importlib_metadata>=1.7.0;python_version<'3.9'
```

Comment:
This might not be needed anymore - I believe Python has that library by default from v3.8

Reply:
Tested with 3.8 and 3.10